system/vmware-monitoring/templates/_helper.tpl

```diff
@@ -34,6 +34,14 @@ prometheus-vmware-{{ $vropshostname._0 | trimPrefix "vrops-" }}
 {{- end -}}
 
 {{- define "maiaFederationMatches" -}}
-{{- range $.Values.maiaFederation.matches -}}.*{{- . | trimPrefix "vrops_virtualmach" -}}|
+vrops_virtualmachine_(?:
+{{- range .Values.maiaFederation.matches -}}
+{{- . | trimPrefix "vrops_virtualmachine_" -}}|
```

Comment:
I would prefer having at least 3 letters in place to avoid duplication with other metrics like we had it in before? What's the reason for dropping the whole string? 
```suggestion
{{- . | trimPrefix "vrops_virtualmach" -}}|
```

Something like `cpu_contention_ratio` could be host based too.

Reply:
This is the query with the additional hostsystem metrics:

`{__name__=~"vrops_virtualmachine_(?:config_hardware_memory_kilobytes|cpu_contention_ratio|cpu_demand_ratio|cpu_usage_ratio|cpu_usage_average_mhz|cpu_wait_summation_miliseconds|datastore_outstanding_io_requests|disk_usage_average_kilobytes_per_second|diskspace_gigabytes|diskspace_virtual_machine_used_gigabytes|guest_os_full_name_info|guest_tools_version_info|memory_active_ratio|memory_activewrite_kilobytes|memory_balloning_ratio|memory_consumed_kilobytes|memory_contention_ratio|memory_usage_average|network_data_received_kilobytes_per_second|network_data_transmitted_kilobytes_per_second|network_packets_dropped.+|network_packets.+|network_usage_average_kilobytes_per_second|number_vcpus_total|oversized.+|runtime_connectionstate|runtime_powerstate|summary_ethernetcards|swapin_memory_kilobytes|swapped_memory_kilobytes|undersized.+|virtual_disk_average.+|virtual_disk_read_kilobytes_per_second|virtual_disk_write_kilobytes_per_second|)|vrops_hostsystem_(?:cpu_model|cpu_sockets_number|cpu_usage_average_percentage|memory_ballooning_kilobytes|memory_contention_percentage|)",project!~"internal",vccluster!~".*management.*"}`

Reply:
Especially this part: `vrops_hostsystem_(?:cpu_model|cpu_sockets_number|cpu_usage_average_percentage|memory_ballooning_kilobytes|memory_contention_percentage|)` should make it clear how it works.

Reply:
Guess it's not so odd anymore.

Reply:
alright, got it. we go for it, thanks!